src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_upscale.py

```diff
@@ -90,6 +91,25 @@ def __init__(
     ):
         super().__init__()
 
+        # check if vae has a config attribute `scaling_factor` and if it is set to 0.08333, else set it to 0.08333 and deprecate
+        is_vae_scaling_factor_set_to_0_08333 = (
+            hasattr(vae.config, "scaling_factor") and vae.config.scaling_factor == 0.08333
+        )
+        if not is_vae_scaling_factor_set_to_0_08333:
+            deprecation_message = (
+                "The configuration file of the vae has  set the configuration `scaling_factor` to"
+                f" {vae.config.scaling_factor}. which seems highly unlikely. If your checkpoint is a fine-tuned"
+                " version of any of the following: \n- stabilityai/stable-diffusion-x4-upscaler \n you should change"
+                " 'scaling_factor' to 0.08333 in the configuration file. Please make sure to update the config"
+                " accordingly as not setting `scaling_factor` in the config might lead to incorrect results in future"
+                " versions. If you have downloaded this checkpoint from the Hugging Face Hub, it would be very nice"
+                " if you could open a Pull request for the `vae/config.json` file"
```

Comment:
```suggestion
                "The configuration file of the vae does not contain `scaling_factor` or it is set to"
                f" {vae.config.scaling_factor}, which seems highly unlikely. If your checkpoint is a fine-tuned"
                " version of `stabilityai/stable-diffusion-x4-upscaler` you should change 'scaling_factor' to 0.08333"
                " Please make sure to update the config accordingly, as not doing so might lead to incorrect results"
                " in future versions. If you have downloaded this checkpoint from the Hugging Face Hub, it would be"
                " very nice if you could open a Pull request for the `vae/config.json` file"
```

Reply:
Also for this could you please make a mass PR on the Hub?